### DISCUSSION TITLE
Breaking changes in v7

### DISCUSSION THREAD

**Discussion by JustinBeckwith** [nodejs/undici#3917] [General]
Greetings folks, and congrats on the release!  Usually with a semver major release, there's some kind of breaking change.  Looking at the release notes for v7 ... it's very difficult to sus out if there were any such changes.  

Was anything breaking between v6 and v7?  Anything we should be looking out for?

**Comment by metcoder95** [comment]
You can use this post from @mcollina as reference: https://bsky.app/profile/nodeland.dev/post/3lbwxbrpey225

**Comment by ethanmick** [comment]
This is a pretty good recap of what's in v7: https://blog.platformatic.dev/undici-v7-is-here

I haven't found an exact upgrade guide yet.

**Comment by mpawlow** [comment]
- Linking in a duplicate discussion: https://github.com/nodejs/undici/discussions/4047
